### ISSUE TITLE
negative queue count

### ISSUE THREAD

**Issue by fgregg** [datamade/dedupe-api#91]
![screenshot from 2015-02-18 10 33 42](https://cloud.githubusercontent.com/assets/536941/6251465/b8d04efe-b759-11e4-9030-f8ed5629cfee.png)

**Comment by fgregg** [comment]
I think this has been fixed.

**Comment by fgregg** [comment]
Nope, this has not been fixed.

**Comment by fgregg** [comment]
```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.4/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.4/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/fgregg/public/dedupe-api/api/auth.py", line 123, in decorated
    return f(*args, **kwargs)
  File "/home/fgregg/public/dedupe-api/api/review.py", line 301, in mark_canon_cluster
    machine.label(entity_id, 1)
  File "/home/fgregg/public/dedupe-api/api/utils/review_machine.py", line 40, in label
    self.weight = rlr.lr(labels, attributes, 11)
  File "rlr/lr.pyx", line 24, in rlr.lr.lr (rlr/lr.c:1385)
    
IndexError: index 0 is out of bounds for axis 0 with size 0
```

**Comment by evz** [comment]
Closed with 486f27860819c9cbe22ef94563d3da5e22b695ae
